save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/CommentDto.kt

```diff
@@ -0,0 +1,25 @@
+package com.saveourtool.save.entities
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property message
+ * @property userName
+ * @property userAvatar
+ * @property section
+ */
+@Serializable
+data class CommentDto(
+    val message: String,
+    val userName: String,
+    val userAvatar: String?,
+    val section: String = "",
```

Comment:
what section will be used by default? If here we set ""?

Reply:
probably its better to document it in kdoc

Reply:
The backend will check that the Section field is filled. This field in the DTO is only needed to save a comment